internal/runners/packages/packages.go

```diff
@@ -50,7 +50,7 @@ func executePackageOperation(pj *project.Project, out output.Outputer, authentic
 		_, err = model.IngredientByNameAndVersion(name, version, ns)
 	}
 	if err != nil {
-		return locale.WrapError(err, "package_ingredient_err", "Failed to resolve an ingredient named {{.V0}}.", name)
+		return addSuggestions(err, ns, name)
```

Comment:
This is assuming that the error means there are no results. You should assert the error type.